catalog/vpc-policies.yaml

```diff
@@ -0,0 +1,21 @@
+- sid: "DenyVpcDeletingFlowLogs"
+  effect: "Deny"
+  actions:
+    - "ec2:DeleteFlowLogs"
+    - "logs:DeleteLogGroup"
+    - "logs:DeleteLogStream"
+  resources:
+    - "*"
+
+- sid: "DenyVpcInternetAccess"
+  effect: "Deny"
+  actions:
+    - "ec2:AttachInternetGateway"
+    - "ec2:CreateInternetGateway"
+    - "ec2:CreateEgressOnlyInternetGateway"
+    - "ec2:CreateVpcPeeringConnection"
+    - "ec2:AcceptVpcPeeringConnection"
+    - "globalaccelerator:Create*"
+    - "globalaccelerator:Update*"
```

Comment:
why it's related to creating/updating of `globalaccelerator`?
Global Accelerator itself does not need a VPC